openslides/core/static/js/core/start.js

```diff
@@ -46,6 +47,27 @@ angular.module('OpenSlidesApp.core.start', [])
                 this.bootup();
             },
         };
+
+        // We need to 'ping' the server with a get request to whoami, because then we can decide,
+        // if the server is down or respond with a 403 (this cannot be differentiated with websockets)
+        autoupdate.registerRetryConnectCallback(function () {
+            return $q(function (resolve) {
```

Comment:
But in this case I sometimes want to reject in the success callback of that request. I tested returning true/false, but this doesn't work for me..